modular_skyrat/modules/customization/modules/culture/faction/faction_generic.dm

```diff
@@ -0,0 +1,143 @@
+	Gateway installations and planets. Somewhat unscrupulous, FTU stations and merchants are known for \
+	selling a great many restricted items and supplying all manner of goods indiscriminately."
+	economic_power = 1.3
+
+/datum/cultural_info/faction/fleet
+	name = "SCG Fleet"
+	description = "The Fleet is the primary component of the SCG Defence Forces. It mostly protects SCG space from pirates, smugglers, \
+	and occasional vox raiders, with its last serious military engagement being the Gaia war over 10 years ago. \
+	The oldest component of the Defence Forces, the Fleet considers itself the pinnacle of Human military accomplishment. \
+	It is well funded, but seen as complacent and bloated. It is made up of five Fleets, with missions ranging from \
+	diplomatic security for Sol dignitaries, to border patrol along the vast frontier and quick reaction force duties. \
+	Any SCG human citizen with at least a secondary school level of education can enlist into the Fleet, going through \
+	basic training and on-ship training for their rating (speciality). Commissioned officer candidates require at least a \
+	Bachelor degree and go through a lengthy training process before being assigned to a warship. \
+	Recently Sol-made IPCs were also permitted to enlist, provided they agree to install a law shackle."
+
+/datum/cultural_info/faction/torchco
+	name = "Expeditionary Corps Organisation"
+	description = "The Expeditionary Corps Organisation is a government-owned, publically traded limited liability company assembled from the corporate \
+	backers who asssisted in funding the Torch project when it opened five years ago. Major players include NanoTrasen, Xynergy and \
+	Hephaestus Industries, as well as dozens of others. The Expeditionary Corps Organisation reports its earnings and operation plans directly to a board of \
+	directors, made up of corporate representatives and members of Expeditionary Corps command and the Committee for the Economy. \
+	While on the surface the Expeditionary Corps Organisation appears to have noble goals, it is in reality a compromise between the noble ideals of pioneering \
+	science, and the corporate desire to make a quick buck off whatever is found. Rumours also abound about shady practices, hidden \
+	protocols, and xenoartifacts being 'lost' in transit, but nothing has been conclusively proven as of yet, and SCG doubtless is \
+	reluctant to harpoon their own project."
+	economic_power = 1.2
+
+/datum/cultural_info/faction/gcc
+	name = "Gilgamesh Colonial Confederation"
+	description = "The Independent Colonial Confederation of Gilgamesh, commonly referred to as the Gilgamesh Colonial Confederation (GCC), is a regional power in human space,\
+	bordered by the Sol Central Government towards the galactic centre. The capital of the Confederation is the city of Ameranth on the planet of Terra \
+	in the Gilgamesh system. Externally heavily militant and isolationist, the GCC, internally, is heavily libertarian, with \
```

Comment:
```suggestion
	in the Zaurghis system. Externally heavily militant and isolationist, the CCC, internally, is heavily libertarian, with \
```

Reply:
The Giglamesh system, as well as the ICCG are both in the code already, too, in the description for the Russian language, weh.